src/components/AttachmentPicker/index.native.js

```diff
@@ -39,15 +44,15 @@ const documentPickerOptions = {
  */
 function showPermissionsAlert() {
     Alert.alert(
-        'Camera Permission Required',
-        'Expensify.cash does not have access to your camera, please enable the permission and try again.',
+        translate(ONYXKEYS.PREFERRED_LOCALE, 'cameraPermissionRequired'),
+        translate(ONYXKEYS.PREFERRED_LOCALE, 'expensifyDoesntHaveAccessToCamera'),
```

Comment:
I think this is wrong:
1. Why is it using translate directly and not the translate from withLocalize? 
2. Per comment above, `ONYXKEYS.PREFERRED_LOCALE` is the name of the key, not the value

(this applies to all calls in this file that are not using this.props.translate)

Reply:
> Why is it using translate directly and not the translate from withLocalize?

Because translate from withLocalize can only be accessed from component, not from function.
You [suggested](https://github.com/Expensify/Expensify.cash/issues/2408#issuecomment-823479714) calling translate method directly in such situations

> Per comment above, ONYXKEYS.PREFERRED_LOCALE is the name of the key, not the value

How can I call translate method directly with this key then? withOnyx also can be used only in components 

Thanks

Reply:
> Because translate from withLocalize can only be accessed from component, not from function.
> You suggested calling translate method directly in such situations

Yes, but you still need to get the correct locale from Onyx manually inn those cases

> How can I call translate method directly with this key then? withOnyx also can be used only in components

You need to call `Onyx.connect` and pass `ONYXKEYS.PREFERRED_LOCALE ` as the `key`